### ISSUE TITLE
batch normalization

### ISSUE THREAD

**Issue by zhly0** [bioinf-jku/SNNs#1]
@bioinf-jku,thank you for your nice work!
I am a new to deep learning,and I have some simply questions,since the net in your test code is not very deep,It makes no big different to add batch normalization layers after each convolution layers,but if the net is very deep,is it necessary to add batch normalization layer after each convolution layers?Or there is no need to do so since the activation function selu has the ability to batch normalize the input layer?
thank you in advance!

**Comment by gklambauer** [comment]
Hello jhly0, 
Yes, you are right that in the tutorial cases that we present the architectures are not too deep. For fully-connected networks, batchnorm is not necessary. For CNNs, there are people who left batchnorm in as a regularizer. There is a discussion about this on reddit: https://www.reddit.com/r/MachineLearning/comments/6g5tg1/r_selfnormalizing_neural_networks_improved_elu/?st=j3wrdmix&sh=d2824d1a
Regards.

**Comment by zhly0** [comment]
@gklambauer thanks for reply!
